src/Polysemy/IO.hs

```diff
@@ -37,16 +38,16 @@ import Polysemy.Internal.Union
 runIO
     :: forall m r a
      . ( MonadIO m
-       , Member (Lift m) r
+       , Member (Embed m) r
        )
-    => Sem (Lift IO ': r) a
+    => Sem (Embed IO ': r) a
     -> Sem r a
-runIO = interpret $ sendM . liftIO @m . unLift
+runIO = runEmbed $ liftIO @m
 {-# INLINE runIO #-}
 
 
 ------------------------------------------------------------------------------
--- | Given some @'MonadIO' m@, interpret all @'Lift' m@ actions in that monad
+-- | Given some @'MonadIO' m@, interpret all @'Embed' m@ actions in that monad
```

Comment:
Let's rename this to `runEmbeddedInIO`

Reply:
Ah, I didn't see this before commenting above.